app/src/androidTest/java/woowacourse/payments/ui/CardPasswordTextFieldTest.kt

```diff
@@ -0,0 +1,72 @@
+package woowacourse.payments.ui
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertTextEquals
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import woowacourse.payments.ui.payments.CardPasswordTextField
+
+class CardPasswordTextFieldTest {
+    @get:Rule
+    val composeTestRule = createComposeRule()
+
+    @Before
+    fun setup() {
+        composeTestRule.setContent {
+            var cardPassword by remember { mutableStateOf("") }
+            CardPasswordTextField(cardPassword = cardPassword) { newValue ->
+                cardPassword = newValue
+            }
+        }
+    }
+
+    @Test
+    fun `비밀번호는_숫자만_입력_가능해야_한다`() {
+        // when
+        val textField = composeTestRule.onNodeWithTag("CardPasswordTextField")
+        textField.performTextInput("1")
+        textField.performTextInput("a")
+        textField.performTextInput("2")
```

Comment:
```suggestion
        textField.performTextInput("1a2")
```
이렇게 하나로 묶어볼 수도 있겠군요 !

Reply:
입력이 숫자만으로 이루어져있는지 판단하기 위해 `isDigitsOnly()` 메서드를 사용하고 있어서 "1a2" 라는 문자가 한 번에 들어오게 되면 잘못된 입력으로 인식해서 return 하게 구현되어있어요 !

모두가 쉽게 이해할 수 있는 코드가 되려면 `isDigitsOnly()` 메서드 대신 입력 문자열을 순회하면서 숫자인 경우만 결합시키는 방식으로 수정되어야 할 것 같네요!